src/components/Icon/ColonIcon.tsx

```
@@ -0,0 +1,20 @@
+const ColonIcon = ({ size = 24, className }: IconProps) => {
+  return (
+    <svg
+      width={size}
+      height="112"
+      viewBox="0 0 24 112"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      className={className}
+      aria-label="콜론"
+    >
+      <circle cx="12" cy="12" r="12" fill="currentColor" />
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)